records_mover/records/types.py

```diff
@@ -46,6 +46,7 @@ class UrlDetailsEntry(TypedDict):
     from typing_extensions import Literal  # noqa
 
     from mypy_extensions import TypedDict
+    # TODO: Can these use the types below?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker